src/sb_sw_lib_tests.c.h

```diff
@@ -1620,11 +1620,105 @@ _Bool sb_test_sw_invalid_scalar(void)
     return 1;
 }
 
+// Test that sweet-b is not vulnerable to the "Psychic Paper" attack.
+// More specifically, verify that for some signature (r, s) if r or s = 0
+// signature verification returns invalid.
```

Comment:
Still missing the requested comment in `sb_sw_lib.c` indicating that the check on zero scalars is tested in this test case.